platform/app-manager/code-container/oracle/index.js

```diff
@@ -1,17 +1,19 @@
-const fs = require("fs");
+const fs = require("fs-extra");
 const path = require("path");
 const mqtt = require("mqtt");
 
-const mqttTopic = 'gateway-data';
-var callbackMap = {};
-var initialized = false;
+let mqttTopic = undefined;
+let callbackMap = {};
 
 function __initialize() {
 	const metadataFilePath = path.join(__dirname, '../metadata.json');
+	// The topic will be customized after sensor-stream-manager completed
```

Comment:
Add a "TODO:"
Makes it easier to search through all todos later on in the IDE. There is a TODO tab in WebStorm where you can see all TODOs of your project.